src/components/Retip.vue

```diff
@@ -57,31 +57,16 @@
           An error occurred while sending retip
         </div>
         <div v-if="!showLoading">
-          <div class="input-group">
-            <input
-              v-model.number="value"
-              type="number"
-              min="0"
-              step="0.1"
-              class="form-control retip__value"
-              aria-label="Default"
-            >
-            <div class="input-group-append">
-              <span class="input-group-text append__ae">
-                <span class="ae">AE</span>
-                <fiat-value
-                  :display-symbol="true"
-                  :amount="value.toString()"
-                />
-              </span>
-            </div>
-            <ae-button
-              :disabled="!isDataValid"
-              @click="retip()"
-            >
-              Retip
-            </ae-button>
-          </div>
+          <ae-input-amount
+            v-model.number="value"
+            :amount="value.toString()"
+          />
```

Comment:
```suggestion
          <ae-input-amount v-model="value" />
```
why don't do like that?